inlong-sort/sort-dist/pom.xml

```diff
@@ -80,6 +82,11 @@
             <artifactId>sort-format-json</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>sort-format-json-v1.5</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
` sort-format-json` and `sort-format-json-v1.5`will cause conflict with v1.13 when package -pv1.1.3.

Reply:
Profile will distinguish between packaging.